crates/topos-tce-storage/src/validator/mod.rs

```diff
@@ -467,12 +505,19 @@ impl WriteStore for ValidatorStore {
             _ = self.pending_tables.pending_pool.delete(&pending_id);
         }
 
-        if let Ok(Some(certificate)) = self
+        if let Ok(Some(next_certificate)) = self
             .pending_tables
             .precedence_pool
             .get(&certificate.certificate.id)
         {
-            self.insert_pending_certificate(&certificate)?;
+            debug!(
+                "Delivered certificate {} unlocks {} for broadcast",
```

Comment:
```suggestion
                "Delivered certificate {}. This unlocks {} for broadcast",
```

Reply:
I think I'll keep the PR format as I was using it really often when debug just by doing ` grep 'certA unlocks certB'`.